### PR TITLE
fix(webapp): template from global is modifiable

### LINKED ISSUE #3061: Bug: Custom templates created from global templates are not editable/deletable after creation
## Steps to reproduce the behavior

1. Go to a board created from any of the global templates
2. Click on the "..." sidebar menu on a board name
3. Select  "New template from board" option
4. Name the custom template
5. Go to template picker
6. Mouse over the new custom template
7. Try to edit or delete template
8. See error - no buttons to edit or delete

Note: Issue does not occur for custom templates created from scratch directly from the template picker, or when using the sidebar "New template from board" option on a template created from scratch. Only applies to creating a custom template from a global template.

## Expected behavior

Custom templates created from a global template should display the edit and delete buttons when mousing over the template name. 

## Screenshots (optional)

https://user-images.githubusercontent.com/93531870/168148346-cd11341b-44ee-48b4-bb58-f1fe8544d059.mov

## Edition and Platform

 - Edition: Mattermost Boards (plugin)
 - Version: v7.0
 - Browser and OS: Chrome on Mac

Related to #3038

### PR DESCRIPTION
**Pull request by Pinjasaur**
#### Summary

Custom templates created from global templates are now able to be edited/deleted in the UI.

#### Ticket Link

Fixes #3061 

#### Screencap

https://user-images.githubusercontent.com/6335792/170309728-685a5b18-a137-4386-9751-efafdd318835.mov

### CODE REVIEW

**Comment by Pinjasaur** [comment]
@chenilim @wuwinson does this need to be cherrypicked to `main` too? unsure offhand based on all the permissions changes

**Comment by sbishel** [comment]
@Pinjasaur - You should test it in main and see if it is an issue. If so, it should also be fixed there. I don't think a CP will work because of the database changes.

File: webapp/src/mutator.ts [hunk 1]
```diff
@@ -795,6 +795,7 @@ class Mutator {
         } else if (asTemplate) {
             // Template from board
             newBoard.title = 'New board template'
+            newBoard.fields.templateVer = 0 // default it to display edit/delete actions
```
[the remark this answers is not in the document]
Comment (Pinjasaur) [reply webapp/src/mutator.ts:798]: Yeah, I considered that initially but it seemed like mirroring the behavior in `createBoard` would be more consistent: https://github.com/mattermost/focalboard/blob/856120ffbdd855ebcf39ed17467aaa1eb785feb2/webapp/src/blocks/board.ts#L70

**Comment by Pinjasaur** [comment]
> @Pinjasaur - You should test it in main and see if it is an issue. If so, it should also be fixed there. I don't think a CP will work because of the database changes.

@sbishel behavior is as-expected in `main` when testing locally.

File: webapp/src/mutator.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (sbishel) [reply webapp/src/mutator.ts:798]: Ok...I'll go with that...

**Review by sbishel — APPROVED** [review]

**Pull request merged**